server/container_stats.go

```diff
@@ -1,12 +1,22 @@
 package server
 
 import (
+	"path/filepath"
+
 	"github.com/cri-o/cri-o/internal/oci"
+	crioStorage "github.com/cri-o/cri-o/utils"
+	"github.com/sirupsen/logrus"
 	"golang.org/x/net/context"
 	pb "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
 )
 
 func buildContainerStats(stats *oci.ContainerStats, container *oci.Container) *pb.ContainerStats {
+	// TODO: Fix this for other storage drivers. This will only work with overlay.
```

Comment:
does it need to check overlay is used here and in case it is not leave `WritableLayer = nil`?

Reply:
@giuseppe do you know where I can check if the overlay is used?

Reply:
Yes, via the `server.Config().Storage` variable, the function just has to be bound to the `*Server` instance.